webrtc/libjingle/xmpp/boshsocket.cc

```diff
@@ -155,7 +156,9 @@ namespace buzz {
 #ifdef FEATURE_ENABLE_SSL
     if (tls_ != buzz::TLS_DISABLED) {
       first_socket = rtc::SSLAdapter::Create(first_socket);
+      ((rtc::SSLAdapter*)first_socket)->set_ignore_bad_cert(true);
```

Comment:
Is this just for Fiddler?  If so, can we block it with PSY_PRODUCT or something?